07/28/2024 10:20:37 AM Logging starts
Traceback (most recent call last):
  File "./popular_projects_snippets_dataset/flask/bodies/body_345_guide_1_3.py", line 17, in <module>
    root_mod_name, _, _ = import_name.partition(".")
NameError: name 'import_name' is not defined
Command exited with non-zero status 1
1.51user 4.86system 0:00.64elapsed 992%CPU (0avgtext+0avgdata 108120maxresident)k
0inputs+16outputs (0major+25115minor)pagefaults 0swaps
